src/components/tasks/card/index.tsx

```diff
@@ -64,6 +64,18 @@ const Card: FC<Props> = ({
         }
     }, [state?.isLoggedIn]);
 
+    useEffect(() => {
```

Comment:
Using side-effects for this feels like bad design. Can we improve it using a global store like mentioned above?

Reply:
Worth exploring, what if we put the URL directly in the `img` tag?

Reply:
@hjaintech I believe the Tasks API only returns `userIds`. The user data has to be fetched separately via the UsersAPI to get profile picture for the user.
So, from the list of tasks, we need to find the list of unique userIds, and fetch user data for each userId via Users API.